meson_options.txt

```diff
@@ -120,6 +120,9 @@ option('pamconfdir', type : 'string',
 
 option('fallback-hostname', type : 'string', value : 'localhost',
        description : 'the hostname used if none configured')
+option('gateway-hostnames', type : 'combo',
+       choices : ['', '_gateway', '_gateway,gateway'], value : '_gateway',
+       description : 'hostnames for the synthetic gateway')
```

Comment:
This is a bit hard to understand IMHO. `symbolic name(s) for the default gateway`?